test/unit/omp-target/test-reduce-tupleminloc.cpp

```diff
@@ -87,13 +90,15 @@ class ReductionTupleLocTestTargetOMP : public ::testing::Test
 
   virtual void TearDown()
   {
+    RAJA::Real_ptr *a = array;
+    RAJA::Real_ptr d = data;
     // NOTE: clang prefers cast to void * and fails compilation otherwise.
```

Comment:
Are these comments still valid after this change?

Reply:
Good question, I don't think so, should be able to remove that.

Reply:
To my surprise, it does in fact fail if you get rid of that cast, on clang.  It's a hard error, so I'd keep that in.

Reply:
I didn't investigate why this occurred, but I originally chose the lesser of two evils. (void *) creates compilation warnings in gcc, and compiles without error in clang.